auk-notebook.ipynb

```diff
@@ -691,13 +782,23 @@
     "positions = {x[0]: np.array([x[1],x[2]]) for x in zippos}\n",
     "\n",
     "# Node sizes based on degree.\n",
-    "size = np.array([x[1] * 100 for x in graph.nodes('size')])\n",
+    "size = np.array([x[1] * NETWORK_NODE_SIZE for x in graph.nodes('size')])\n",
     "\n",
     "# Draw the graph.\n",
-    "nx.draw(graph, pos=positions, show_labels=True, labels=mapping, font_size=10, node_size=size, node_color=colormap)\n",
+    "nx.draw(graph, pos=positions, show_labels=NETWORK_SHOW_LABELS, labels=mapping, font_size=NETWORK_FONT_SIZE, node_size=size, node_color=colormap)\n",
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "NETWORK_EGO_NODE_ORDINAL = 0 # Selects the largest node. Use `1` for 2nd largest,`2` for third largest etc.\n",
```

Comment:
And last one. Same thing here, a markdown cell above this, along the lines of:

```
Change the variables in the following cell to manipulate the visualization that you're about to generate. What node do you want the diagram to focus on? The "biggest" one? Or a specific one?
```

Can you provide a bit more info here around what the largest node is. What makes it the largest one?